amr-wind/core/MLMGOptions.H

```diff
@@ -104,6 +104,14 @@ private:
     //! If set to true, AMReX won't abort if linear solver doesn't converge
     //! within max iterations
     bool do_fixed_iters{true};
+
+    //! option to use the nsolve bottom solver
+    //! added to help with overset masking
+    bool do_nsolve{false};
+
+    //! set grid size for nsolve
+    //! use a larger number on GPUs
+    int nsolve_grid_size{16};
```

Comment:
We should set the default depending on whether GPU is enabled or not?

Reply:
I added an #ifdef